DocumentFormat.OpenXml/src/Framework/OpenXmlPackage.cs

```diff
@@ -1384,6 +1384,56 @@ internal class PartUriHelper
             private Dictionary<string, int> _sequenceNumbers = new Dictionary<string, int>(20);
             private Dictionary<string, int> _reservedUri = new Dictionary<string, int>();
 
+            //List of contentTypes that need to have a 1 appended to the name
+            //for the first item in the package
+            private static readonly HashSet<string> _numberedContentTypes = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
They are from the ISO/IEC 29500 spec. I listed the section numbers for reference.